components/Checkbox.js

```diff
@@ -40,8 +42,8 @@ const Checkbox = ({ className, id, labelText, onChange, ...other }) => {
         <span className="bx--checkbox-appearance">
           <Icon
             className="bx--checkbox-checkmark"
-            description="checkmark"
-            alt="checkmark"
+            description={iconDescription}
+            alt={iconDescription}
```

Comment:
could you actually change this into two different props

```
descriptionText
altText
```

with both defaults being the same in this scenario?